### DISCUSSION TITLE
autodiscover-filter include apps/*/* but apps/automation-test/*

### DISCUSSION THREAD

**Discussion by marvinfischenich** [renovatebot/renovate#17154] [Request Help]
Hi guys,
for my bot on GitLab self-hosted i want to to filter the repos.

I want to include `apps/*/*`
but i also want to exclude `apps/automation-test/*`

How can i make that happen?
i tried those in CI variable `RENOVATE_EXTRA_FLAGS` which isn't working:

`--autodiscover-filter=apps/*/* !apps/automation-test/*`

`--autodiscover-filter=/apps\/(?!automation-test).*/`

Then i tried `apps\/(?!automation-test).*` in CI variable `RENOVATE_AUTODISCOVER_FILTER` which didn't work neither.

Thanks for your help!

**Comment by PhilipAbed** [comment]
duplicate of https://github.com/renovatebot/renovate/discussions/16466
